scripts/kind-e2e/e2e.sh

```diff
@@ -56,18 +56,18 @@ test_subctl_gather
 
 # Run subctl diagnose as a sanity check
 
-${DAPPER_SOURCE}/bin/subctl diagnose all
-${DAPPER_SOURCE}/bin/subctl diagnose firewall inter-cluster ${KUBECONFIGS_DIR}/kind-config-cluster1 ${KUBECONFIGS_DIR}/kind-config-cluster2
+subctl diagnose all
+subctl diagnose firewall inter-cluster ${KUBECONFIGS_DIR}/kind-config-cluster1 ${KUBECONFIGS_DIR}/kind-config-cluster2
 
 # Run benchmark commands for sanity checks
 
-${DAPPER_SOURCE}/bin/subctl benchmark latency --intra-cluster ${KUBECONFIGS_DIR}/kind-config-cluster1
+subctl benchmark latency --intra-cluster ${KUBECONFIGS_DIR}/kind-config-cluster1
 
-${DAPPER_SOURCE}/bin/subctl benchmark latency ${KUBECONFIGS_DIR}/kind-config-cluster1 ${KUBECONFIGS_DIR}/kind-config-cluster2
+subctl benchmark latency ${KUBECONFIGS_DIR}/kind-config-cluster1 ${KUBECONFIGS_DIR}/kind-config-cluster2
 
-${DAPPER_SOURCE}/bin/subctl benchmark throughput --intra-cluster ${KUBECONFIGS_DIR}/kind-config-cluster1
+subctl benchmark throughput --intra-cluster ${KUBECONFIGS_DIR}/kind-config-cluster1
 
-${DAPPER_SOURCE}/bin/subctl benchmark throughput --verbose ${KUBECONFIGS_DIR}/kind-config-cluster1 ${KUBECONFIGS_DIR}/kind-config-cluster2
+subctl benchmark throughput --verbose ${KUBECONFIGS_DIR}/kind-config-cluster1 ${KUBECONFIGS_DIR}/kind-config-cluster2
```

Comment:
I think we should remove these checks (L59-70) since they test `subctl` and the `subctl`repo is also doing it.

Reply:
I would say L53-70